docs/release-notes.rst

```diff
@@ -1,3 +1,141 @@
+Release Notes (Version 2.4)
+===========================
+
+General
+-------
+
+**F5 ACI ServiceCenter Upgrade is not supported from Version 1.0 to Version 2.4**
+
+The F5 ACI ServiceCenter application does not have upgrade support from Version 1.0 to Version 2.4. In order to install a new version of the app the steps to be followed are:
+1. Uninstall the existing Version 1.0 of the application from APIC Apps tab.
+2. Install and enable Version 2.4 of the application by downloading it from https://dcappcenter.cisco.com/
+
+Note: App upgrades are supported in version 2.0 and higher
+
+------
+
+**File system convergence**
+
+APIC Filesystem Glusterfs takes 15 to 20 Minutes to recover when APIC cluster goes unhealthy
+
+During the APIC operations like APIC reboot, upgrade, or decommission/recommission, the APIC filesystem needs some time to recover and resume.
+
+Symptoms users may see:
+1. Application operations throw a (sqlite3.OperationalError) disk I/O error
+2. User may not find previously added BIG-IP entries in the application
+3. New Application installation may fail
+
+**Workaround**: Wait for 20 minutes and try to access the application again. The Glusterfs should recover within some time automatically and the application should be accessible again.
```

Comment:
I'd suggest getting rid of "within some time".. so it would be 'should recover automatically'